modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -0,0 +1,136 @@
+[id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
+= Modifying the retention time and size for Prometheus metrics data
+
+By default, Prometheus automatically retains metrics data for 15 days.
+You can modify the retention time to change how soon data is deleted by specifying a time value in the `retention` field.
+You can also configure the maximum amount of disk space used by the retained metrics data by specifying a size value in the `retentionSize` field.
+If this size limit is reached, the oldest data is deleted first until the disk space used by the data is again below the maximum.
+
+Note the following behaviors of these data retention settings:
+
+* The size-based retention policy applies to all data block directories in the `/prometheus` directory, including persistent blocks, write-ahead log (WAL) data, and m-mapped chunks. 
+* Data in the `/wal` and `/head_chunks` directories counts toward the retention size limit, even though data from these directories will never be purged based on size- or time-based retention policies. 
+Thus, if you set a retention size limit lower than the maximum size set for the `/wal` and `/head_chunks` directories, in effect you have configured the system not to retain any data blocks in the `/prometheus` data directories.
+* The size-based retention policy is applied only when Prometheus cuts a new data block, which occurs every two hours and after the WAL contains at least three hours of data.
+* If you do not explicitly define values for either `retention` or `retentionSize`, retention time defaults to 15 days, and retention size is not set.
+* If you define values for both `retention` and `retentionSize`, both values apply.
+If any data blocks exceed either the defined retention time or the defined size limit, then these data blocks are purged.
+* If you define a value for `retentionSize` and do not define `retention`, only the `retentionSize` value applies.
+* If you do not define a value for `retentionSize` and only define a value for `retention`, only the `retention` value applies.
+
+.Prerequisites
+
+* *If you are configuring core {product-title} monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring components that monitor user-defined projects*:
+** You have access to the cluster as a user with the `cluster-admin` role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+** You have created the `user-workload-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `ConfigMap` object:
+** *To modify the retention time and size for the Prometheus instance that monitors core {product-title} projects*:
+.. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
```

Comment:
...in that case:
```suggestion
. Optional: To modify the retention time and size for the Prometheus instance that monitors core {product-title} projects, edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
```

Reply:
Per our Slack conversation, making these structural changes to the procedure is outside the scope of this PR, so I am leaving it as-is